index.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Promise: Portfolio</title>
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css"
+    integrity="sha512-KfkfwYDsLkIlwQp6LFnl8zNdLGxu9YAA1QvwINks4PhcElQSvqcyVLLD9aMhXd13uQjoXtEKNosOWaZqXgel0g=="
+    crossorigin="anonymous" referrerpolicy="no-referrer" />
+  <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+  <!-- first-section -->
+  <div class="wrapper">
+      <div class="header-wrapper">
+        <!-- navbar-section -->
+        <header class="header-section">
+          <span class="nav-span"><a href="#">Promise Okechukwu</a></span>
+          <span><a href="#"><img src="./images/Enabled.png" class="#" alt="#"></a></span>
+         
+          <ul class="nav-links">
+            <li>
+              <span><a href="#">Portfolio</a></span>
+              <span><a href="#">About</a></span>
+              <span><a href="#">contact</a></span>
+              <span><a href="#"><img src="./images/mail.png" alt="mail logo"></a></span>
+            </li>
+          </ul>
+        </header>
+        <!-- header  section -->
+        <section class="first-section">
+          <div class="header-intro">
+            <h1 class="header-text">
+              Hey There.
+              I’m Promise
+            </h1>
+            <h2 class="header-smaller-text"> I am a Software Developer </h2>
+            <p class="header-paragraph">
+              I can help you build a product,
+              feature or website Look through some of my work and experience!
+              If you like what you see and have a project you need coded,
+              don’t hestiate to contact me.
+            </p>
+          </div>
+          <!-- Icons section -->
+          <div class="header-icons">
+            <ul>
+              <li><a href="https://jheart-vic.github.io/My-Portfolio-Website/"><img src="./images/Github.png"
+                    alt="Github logo"></a></li>
+              <li><a href="https://www.linkedin.com/in/victor-chiemerie/"><img src="./images/Linkedin-icon.svg"
+                    alt="Linkedin logo"></a></li>
+              <li><a href="https://www.github.com/promzzykoncepts"><img src="./images/angel-list.svg" alt="Angelist logo"></a></li>
```

Comment:
- Good job so far, but you have one image not displaying properly, kindly fix this and double-check it works on your end then ask for a review. Also, align the images properly just as in the Figma design
- Please insert background image as in the Figma design, requested by the previous reviewer, use the CSS property to background-image for that

##### Your design
![Screenshot from 2022-06-30 10-04-45](https://user-images.githubusercontent.com/58771507/176638205-bcd0faf0-e54d-4653-b6c4-919179f3393f.png)

##### Figma design

![Screenshot from 2022-06-30 10-07-04](https://user-images.githubusercontent.com/58771507/176638708-d0a784e2-d860-4035-aa08-7e1f9d03bddc.png)